### PR TITLE
Rotate Sidebar Ad to Bioinformatics Webinar

### PR DESCRIPTION
**Pull request by SL8onC**
Post-merge checklist:

The master branch is auto-deployed to `dash.plotly.com`.
Once you have merged your PR, wait 5-10 minutes and check dash.plotly.com
to verify that your changes have been made.

- [ ] I understand

If this PR documents a new feature of Dash:

- [ ] Comment on the original Dash issue with a link to the new docs.
- [ ] Reply to any community thread(s) asking for this feature.

If this PR includes a new dataset available at a remote URL:
- [ ] I have added this dataset to the `datasets/` folder
- [ ] I have added a mapping between the remote URL and the filename in the
`datasets/` folder into the `find_and_replace` dict in `dash_docs/tools.py`

If this PR adds an image or animated GIF:
- [ ] This image was saved and referenced locally rather than via an external link

If I introduced a new relative link inside `dcc.Markdown`:
- [ ] I considered whether I could replace the `dcc.Markdown` call with `rc.Markdown`, which will replace relative links with `tools.relpath` internally. Otherwise, I used e.g. `<dccLink href=tools.relpath('/layout') children="the first chapter"/>` instead of `[the first chapter](/layout)` (importing `tools` from `dash_docs`), and set `dangerously_allow_html=true` in the `dcc.Markdown` call.

If I changed the `chapter_index` by removing or relocating a page:
- [ ] I added a redirect in `dash_docs/server.py` from the old URL to the new URL

### CODE REVIEW

**Review by chriddyp — APPROVED** [review]
💃🏼

**Comment by SL8onC** [comment]
@chriddyp @jackparmer @alexcjohnson even got the EDT right this time lol

File: dash_docs/run.py [hunk 1]
```diff
@@ -95,9 +95,9 @@ def create_contents(contents):
 )
 
 DEFAULT_AD = dict(
-   alt="⚾️ It's finally baseball season! Root for your home team! Register today for our Sports Analytics Webinar on April 21st!",
-   src=tools.relpath('https://images.prismic.io/plotly-marketing-website/6fc81948-6832-45b5-9ab7-cf3ca93275b1_sidebar_ads_4-7-21.jpeg?auto=compress,format'),
-   href='https://go.plotly.com/sports-analytics'
+   alt="🧬Alyssa Morrow shows us how to build RNA-Sequencing apps in Python with Dash, IGV.js, and Pileup,js. Register for the May 20 Webinar!",
```
Comment (jackparmer) [inline dash_docs/run.py:98]: I'd change to:

🧬 Learn how to build RNA-Seq data apps in Python & Dash at this May 20 Webinar.

Comment (SL8onC) [reply dash_docs/run.py]: Updated

**Comment by jackparmer** [comment]
I also think that this image doesn't look good in a thumbnail format @angeladaodao 

https://images.prismic.io/plotly-marketing-website/14e0ba73-31c1-4b26-8f47-4437d4e1ba41_sidebar_ads_bioinformatics.jpeg

I would make the thumbnail a screenshot of the apps instead with big text below:

image

You can reduce the text in the image to:

> Build RNA-Seq data apps in Dash
> May 20 Webinar

With less text, you can make the text in the image bigger so that it's more readable.

**Comment by angeladaodao** [comment]
so something like this?
sidebar_ads_2-17

**Comment by jackparmer** [comment]
Perfect

**Pull request merged**